apps/block_scout_web/lib/block_scout_web/templates/layout/app.html.eex

```diff
@@ -46,7 +46,7 @@
       }
       window.onload = applyDarkMode()
     </script>
-    <div class="layout-container" data-page="app-container">
```

Comment:
slightly worried about this? seems like something that might be something used? I dont see any code in this pr that referenced it

Reply:
So, this was something I added in the original PR but didn't end up using.  It's still not being used so I removed it